src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,55 @@
+package duke.command;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The DeadlineCommand class encapsulates the execution of the deadline command from the user.
+ */
+public class DeadlineCommand extends Command {
+    /** The deadline command inputted by the user. */
+    String fullCommand;
```

Comment:
Perhaps it is better to specify the access modifier for fullCommand in this case? I noticed the same issue in several other places too, such as in the DeleteCommand, DoneCommand, EventCommand and a few more classes.